5_databases/week2/homework part1.sql

```diff
@@ -0,0 +1,28 @@
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (title, description, created, updated, due_date, status_id, user_id)
+ VALUES ("Learn web development", "HTML, CSS, JS, React,etc", "2023-11-15 11:14:00", "2023-11-15 11:14:00", "2024-04-30 11:14:00" ,"2", "1");
```

Comment:
There is no user_id field, so I suspect this must be the id. Also pure integers such as 1 and 2 should not have "" around it. "" is for strings. Instead do something like:
`INSERT INTO task (id, title, description, created, updated, due_date, status_id)
VALUES (50, "Learn web development", "HTML, CSS, JS, React, etc.", "2023-11-15 11:14:00", "2023-11-15 11:14:00", "2024-04-30 11:14:00", 2);`

Reply:
Or eventually just keep it as it is, since the homework tasks states to do it like this. But still remove the quote around the integer :)